pkg/controller/backend/backend_test.go

```diff
@@ -6,8 +6,9 @@ import (
 	"testing"
 
 	"github.com/golang/mock/gomock"
-	"github.com/stretchr/testify/assert"
+	tassert "github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"gotest.tools/assert"
```

Comment:
why two different kinds of assert here?

Reply:
`gotes.tools/assert` allows us to `assert.DeepEqual` which is not a top-level function in the `testify/require` package. `assert.DeepEqual` is used extensively in the ex-CLI compose tests as well

Reply:
why do you need `DeepEqual`? i thought `testify/require` used `reflect.DeepEqual` under the hood?